pdns/iputils.cc

```diff
@@ -22,8 +22,11 @@
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
+
 #include "iputils.hh"
-#include <sys/socket.h> 
+
+#include <sys/socket.h>
+#include <boost/format.hpp>
```

Comment:
I'm actually removing that usage in #11293 so we will need to rebase one of these PRs, likely #11293, but that's fine.